### ISSUE TITLE
Microsoft Edge kremling scope bug

### ISSUE THREAD

**Issue by dvnrsn** [CanopyTax/bandicoot#87]
![edge-kremling-scope](https://user-images.githubusercontent.com/9112811/78082492-623e4180-7370-11ea-9da3-cfa552516ef5.gif)

**Comment by dvnrsn** [comment]
This seemed particular to using a class that was defined by a higher kremling scope. In this case you can see the `display: flex` was lost. Once I redefine the classes more local to the relevant components, the bug is resolved.

**Comment by internettrans** [comment]
easy reproduction case for this?

**Comment by internettrans** [comment]
Also should it be moved to https://github.com/CanopyTax/kremling?

**Comment by dvnrsn** [comment]
yeah good question. I think it's a pretty obscure case. Not sure if it's worth keeping around in either. I only created the issue to document it. It seems to be an Edge specific bug that we can't do much about.
